src/pages/data/ds/[...pathItems].js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @author sriram
+ *
+ */
+
+import React from "react";
+import { useRouter } from "next/router";
+import Listing from "../../../components/data/listing/Listing";
+import constants from "../../../constants";
+import NavigationConstants from "../../../components/layout/NavigationConstants";
+
+/**
+ *
+ * Handle routing to /data/ds/*
+ *
+ */
+export default function DataStore() {
+    const router = useRouter();
+    const pathItems = router?.query.pathItems;
```

Comment:
👍 I just hope this `[...pathItems].js` file name place nice on my old desktop 😆 
Also, should this be
```suggestion
    const pathItems = router?.query?.pathItems;
```